src/main/java/com/github/martynagil/drugstoremanagement/service/EmployeeService.java

```diff
@@ -0,0 +1,43 @@
+package com.github.martynagil.drugstoremanagement.service;
+
+import com.github.martynagil.drugstoremanagement.controller.EmployeeDismissalDto;
+import com.github.martynagil.drugstoremanagement.controller.EmployeeDto;
+import com.github.martynagil.drugstoremanagement.model.Employee;
+import com.github.martynagil.drugstoremanagement.model.Shop;
+import com.github.martynagil.drugstoremanagement.repositories.EmployeeRepository;
+import com.github.martynagil.drugstoremanagement.repositories.ShopRepository;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+
+@Service
+public class EmployeeService {
+
+    private EmployeeRepository employeeRepository;
+    private ShopRepository shopRepository;
+
+    public EmployeeService(EmployeeRepository employeeRepository, ShopRepository shopRepository) {
+        this.employeeRepository = employeeRepository;
+        this.shopRepository = shopRepository;
+    }
+
+    public List<Employee> getAllEmployees() {
+        return employeeRepository.findAll();
+    }
+
+    public void addNewEmployee(EmployeeDto employeeDto) {
+        Shop shop = shopRepository.findById(employeeDto.getShopId())
+                .orElseThrow(() -> new EntityNotFoundException());
+        employeeRepository.save(employeeDto.toEntity(shop));
+    }
+
+    public void dismissEmployee(Long employeeId, EmployeeDismissalDto employeeDismissalDto) {
+        Employee employee = employeeRepository.findById(employeeId)
+                .orElseThrow(() -> new EntityNotFoundException());
+        if (employee.getDateOfDismissal().equals(null)) {
```

Comment:
1. porównania z nullem robi się przy pomocy `==`
2. jak już jest ustawiony dismissal to raczej powinien polecieć wyjątek, np. `IllegalStateException`